endpoints/k8s/k8s

```diff
@@ -733,14 +733,15 @@ function build_pod_spec() {
 
     if [ "$type" == "cs" ]; then
 
-        # Client/server user-provided resources
-        set +u
-        if [ ! -z "${resources[$name]}" ]; then
-            echo -e ",${resources[$name]}" >>$json
-        elif [ ! -z "${resources[default]}" ]; then
-            echo -e ",${resources[default]}" >>$json
+        # If user provides pod securityContext
+        if [ "$type" == "cs" ]; then
+            set +u
+            if [ ! -z "${PsecurityContext[$name]}" ]; then
+                echo -e ",${PsecurityContext[$name]}" >>$json
+            elif [ ! -z "${PsecurityContext[default]}" ]; then
+                echo -e ",${PsecurityContext[default]}" >>$json
+            fi
         fi
-        set -u
```

Comment:
Isn't the `set -u` needed right before the above `fi`?

Reply:
It is needed, but right after the if